samples/backups-get-database-operations.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2020 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+async function getDatabaseOperations(instanceId, projectId) {
+  // [START spanner_list_database_operations]
+  // Imports the Google Cloud client library
+  const {Spanner} = require('@google-cloud/spanner');
+  const {google} = require('../protos/protos');
```

Comment:
@olavloite hope it's ok to ask for your help here. Would you happen to know how to do this import without causing a lint error. Or was there meant to be some other type defined somewhere so that it's not necessary to do this import?

Reply:
I think the simplest way to avoid lint error is by adding `node/no-unpublished-require: off` rule in `samples/.eslintrc.yml` file.

Reply:
I figured it out. src/index.ts exports the protos so all I had to do was to require it from the spanner package.